tests/convert.rs

```diff
@@ -49,3 +49,16 @@ fn convert_boids() {
         msl::write_string(&module, options).unwrap();
     }
 }
+
+/// This is a "poor man's snapshot test." The idea is to eventually use insta (#41).
```

Comment:
What is the goal of this snapshot test? Ensure that the serialized form is unchanged?
It could be a bigger promise to make than we are ready to, at this point.
Although it wouldn't be as much of a problem if the saved RON was smaller :) So maybe with the `Handle<>` serialization change it's going to be ok.

Reply:
Fair enough. I replaced this with a round-trip test, which makes a more reasonable promise.